src/lib.rs

```diff
@@ -64,6 +63,51 @@ const BLOBDIR: &str = "blobs/sha256";
 
 const OCI_TAG_ANNOTATION: &str = "org.opencontainers.image.ref.name";
 
+/// Errors returned by this crate.
+#[derive(Error, Debug)]
+#[non_exhaustive]
+pub enum Error {
+    #[error("i/o error")]
+    /// An input/output error
+    Io(#[from] std::io::Error),
+    #[error("serialization error")]
+    /// Returned when serialization or deserialization fails
+    SerDe(#[from] serde_json::Error),
+    #[error("parsing OCI value")]
+    /// Returned when an OCI spec error occurs
+    OciSpecError(#[from] oci_spec::OciSpecError),
+    #[error("parsing OCI value")]
```

Comment:
This should say "Cryptographic error" instead

Reply:
Good catch, thanks! Fixed